lua/astrocommunity/code-runner/toggletasks-nvim/init.lua

```diff
@@ -0,0 +1,15 @@
+return {
+  "jedrzejboczar/toggletasks.nvim",
+  dependencies = {
+    { "nvim-lua/plenary.nvim" },
+    { "akinsho/toggleterm.nvim" },
+    { "nvim-telescope/telescope.nvim" },
```

Comment:
```suggestion
    { "nvim-telescope/telescope.nvim" },
    {
      "AstroNvim/astrocore",
      opts = {
        mappings = {
          n = {
            ["<Leader>tss"] = { "<Cmd>Telescope toggletasks spawn<CR>", desc = "toggletasks: spawn" },
            ["<Leader>tsc"] = { "<Cmd>Telescope toggletasks select<CR>", desc = "toggletasks: select" },
            ["<Leader>tse"] = { "<Cmd>Telescope toggletasks edit<CR>", desc = "toggletasks: edit config" },
          },
        },
      },
    },
```

map keys with AstroCore